README.md

```diff
@@ -0,0 +1,65 @@
+# ArrayAttributes
+
+A Rails plugin to encapsulate common array attributes methods.
+
+This gem can be used to avoid repeating the same methods in models when working with arrays.
+
+## Support
+
+**This branch targets Ruby 2.4+ only**
+
+
+## Usage
+
+#### Install gem:
+
+```ruby
+gem 'array_attributes'
```

Comment:
isn't it `gem install array attributes`? if you are referring it to the gemfile, I would specify that. Like:
```markdown
Add this line to your  Gemfile:
gem 'array_attributes'
then run:
bundle install
```

Reply:
Yes, you're right